README.md

```diff
@@ -185,7 +185,7 @@ export class BookService {
 
 ## Metric Service
 
-[OpenTelemetry Metrics](https://www.npmjs.com/package/@opentelemetry/api-metrics) allow a user to collect data and export it to metrics backend like Prometheus.
+[OpenTelemetry Metrics](https://www.npmjs.com/package/@opentelemetry/api) allow a user to collect data and export it to metrics backend like Prometheus.
```

Comment:
This implies you are not using OpenTelemetryMetrics anymore, but `OpenTelemetry API`

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/3374